python/graphstorm/dataloading/dataloading.py

```diff
@@ -961,6 +995,8 @@ def fanout(self):
 
 class GSgnnNodeDataLoader(GSgnnNodeDataLoaderBase):
     """ Minibatch dataloader for node tasks
+    
+    GSgnnNodeDataLoader samples GraphStorm node dataset into an iterable over mini-batches of samples.
```

Comment:
Could you also add what the DataLoader will return for iteration as you did for the other two DataLoader?